crates/meroctl/src/cli/auth.rs

```diff
@@ -35,12 +36,15 @@ pub async fn authenticate(api_url: &Url) -> Result<JwtToken> {
 
     let auth_url = build_auth_url(api_url, callback_port)?;
 
+    let info_msg = format!("Trying to open browser to start authentication");
+    InfoLine(&info_msg).report();
```

Comment:
`Report::report` should only be called when `--output-format human`, not unconditionally